File: src/main/java/com/enzo/testemuralis/services/viacep/adapter/ViacepGateway.java
package com.enzo.testemuralis.services.viacep.adapter;

import com.enzo.testemuralis.services.viacep.dtos.EnderecoViacep;

public interface ViacepGateway {
    EnderecoViacep buscarEnderecoPorCep(String cep);
} 

File: dotnet/TesteMuralis/Services/Viacep/Adapter/ViacepGateway.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using TesteMuralis.Services.Viacep.Dto;

namespace TesteMuralis.Services.Viacep.Adapter
{
    public interface ViacepGateway
    {
        EnderecoViacep BuscarEnderecoPorCep(String cep);
    }
}